src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
+
+import DocumentationView from './views/DocumentationView';
+import TimersView from './views/TimersView';
+import ConfigureView from './views/ConfigureView';
+import AppContext from './AppContext';
+
+const Nav = () => {
+  return (
+    <nav>
+      <ul>
+        <li>
+          <Link to="/">Sample Timers</Link>
```

Comment:
The `/` route should contain a list of the timers that the user has added at `/add` to the workout